docs/reference/configuration.md

```diff
@@ -112,8 +112,8 @@ This reference uses `.` to denote the nesting of values.
 + `email.password` `(string)` - A string representing the password of the account. It's recommended to provide this value at runtime from a secret vault of some sort.
 + `email.local_name` `(string)` - A string representing the hostname sent to the SMTP server with the HELO command. By default, "localhost" is sent.
 + `email.sender` `(string)` - A string representing the email account which is set as the sender.
-+ `email.limit_every_duration` `(duration : 1m)` - Average time to wait between sending emails. The zero value means no duration and therefore no emails can be sent.
-+ `email.burst` `(int: 10)` - Maximal burst of emails before applying `limit_every`.
++ `email.limit_every_duration` `(duration : 1m)` - Average time to wait between sending emails. The zero value means *infinite* duration and therefore *unlimited* emails can be sent.
++ `email.burst` `(int: 10)` - Maximal burst of emails before applying `limit_every`. The zero value means no burst and therefore no emails can be sent.
```

Comment:
What is limit_every?

Reply:
It's referring to `limit_every_duration`  (in the previous line)